sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentAnalysisClient/DocumentAnalysisClientLiveTests.cs

```diff
@@ -1146,6 +1146,58 @@ public async Task StartAnalyzeDocumentLayoutWithSelectionMarks()
 
         #endregion
 
+        #region Read
+
+        [RecordedTest]
+        [TestCase(true)]
+        [TestCase(false)]
+        public async Task StartAnalyzeDocumentCanReadPageAndLanguage(bool useStream)
+        {
+            var client = CreateDocumentAnalysisClient();
+            AnalyzeDocumentOperation operation;
+
+            if (useStream)
+            {
+                using var stream = DocumentAnalysisTestEnvironment.CreateStream(TestFile.InvoicePdf);
+                using (Recording.DisableRequestBodyRecording())
+                {
+                    operation = await client.StartAnalyzeDocumentAsync("prebuilt-read", stream);
+                }
+            }
+            else
+            {
+                var uri = DocumentAnalysisTestEnvironment.CreateUri(TestFile.InvoicePdf);
+                operation = await client.StartAnalyzeDocumentFromUriAsync("prebuilt-read", uri);
+            }
+
+            await operation.WaitForCompletionAsync();
+            Assert.IsTrue(operation.HasValue);
+
+            AnalyzeResult result = operation.Value;
+
+            ValidateAnalyzeResult(
+                result,
+                "prebuilt-read",
+                expectedFirstPageNumber: 1,
+                expectedLastPageNumber: 1);
+
+            DocumentPage page = result.Pages.Single();
+
+            // The expected values are based on the values returned by the service, and not the actual
+            // values present in the form. We are not testing the service here, but the SDK.
+
+            Assert.AreEqual(LengthUnit.Inch, page.Unit);
+            Assert.AreEqual(8.5, page.Width);
+            Assert.AreEqual(11, page.Height);
+            Assert.AreEqual(0, page.Angle);
+            Assert.AreEqual(18, page.Lines.Count);
+            Assert.IsEmpty(result.Tables);
+
+            Assert.IsNotEmpty(result.Languages);
```

Comment:
I wondered if there's anything relevant we can validate in any `DocumentLanguage`. We already are doing some basic sanity checks in `ValidateAnalyzeResult` we called above (check confidence being in the [0, 1] interval, null-checks).